cartridge-cli.lua

```diff
@@ -3373,6 +3391,7 @@ function cmd_pack.parse(cmd_args)
             download_token = 'string',
             tag = 'string',
             from = 'string',
+            build_base = 'string',
```

Comment:
build_base_image/build_base_dockerfile/...?

Reply:
I sure that CLI options should be as short as it's possible.

`--build-base` is short, but `base-build-dockerfile` is more informative. 

Maybe `--build-from` is better (similar to `--from`).

Reply:
> Maybe --build-from

IMO, yes